Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -262,9 +262,19 @@ Another advantage in DSR mode is that the client's source IP is preserved, so po
 can match on it at the backend node. In the SNAT mode this is not possible.
 Given a specific backend can be used by multiple services, the backends need to be
 made aware of the service IP/port which they need to reply with. Therefore, Cilium
-encodes this information as an IPv4 option or IPv6 extension header at the cost of
-advertising a lower MTU. For TCP services, Cilium only encodes the service IP/port
-for the SYN packet.
+encodes this information in a Cilium-specific IPv4 option or IPv6 extension header
```

Comment:
nit: in the case of IPv6, we use Destination Option extension header.

Reply:
Will add.